addons/account/models/account_move.py

```diff
@@ -4106,14 +4106,13 @@ def _get_all_reconciled_invoice_partials(self):
                 FROM account_partial_reconcile part
                 JOIN account_move_line debit_line ON debit_line.id = part.debit_move_id
                 WHERE debit_line.move_id IN %s AND part.credit_move_id IN %s
-            '''
-            self._cr.execute(query, [tuple(exchange_move_ids), tuple(counterpart_line_ids)] * 2)
+            ''', *params)
```

Comment:
As above, the accidental complexity of passing `*params` has no added value.  I suggest to replace the `%s` in the query by `%(credit_move_ids)s` and `%(debit_line_ids)s` and pass the corresponding named arguments.  This will greatly improve the readability of the query.